lerobot/scripts/control_robot.py

```diff
@@ -213,8 +225,10 @@ def teleoperate(robot: Robot, fps: int | None = None, teleop_time_s: float | Non
             break
 
 
-def record_dataset(
+def record(
```

Comment:
Here too, this function is 360+ lines and it's very hard to get an overview of what's going on at a high level. I would suggest a refactor or adding a TODO.

Reply:
```python
# TODO(rcadene): Clean this function via decomposition in higher level functions
```